manual/toggl/toggl.nuspec

```diff
@@ -3,13 +3,12 @@
   <metadata>
     <id>toggl</id>
     <title>Toggl Desktop</title>
-    <version>7.1.145</version>
-    <authors>Apprise LLC</authors>
+    <version>7.3.318</version>
+    <authors>Toggl OÜ</authors>
```

Comment:
Ah. Well that's their legal name per their TOS (https://www.toggl.com/legal/terms see the Supplier definition). They're based in Estonia.